src/plugins/WikiRouterPlugin/MediaWikiRouter.ts

```diff
@@ -0,0 +1,34 @@
+import WikiRouter from './WikiRouter';
+import { MwUtilGetUrl } from '@/@types/mediawiki';
+import {
+	inject,
+	InjectionKey,
+} from 'vue';
+
+export default class MediaWikiRouter implements WikiRouter {
+
+	private readonly getUrl: MwUtilGetUrl;
+
+	public constructor(
+		getUrl: MwUtilGetUrl,
+	) {
+		this.getUrl = getUrl;
+	}
+
+	public getPageUrl( title: string, params?: Record<string, unknown> ): string {
```

Comment:
Does this method need to be public? I can't see where it would be used besides `goToTitle`

Reply:
Not for now, though we will probably need it later. I'll make it private, that way I also can write one less test :raised_hands: